### PR TITLE
updated transportation-data URLs

### LINKED ISSUE #328: Update transportation-data references on dashboards
A number of dashboards pull data from `transportation-data.austintexas.io`. These URLs need to be updated to use this base url: `http://transportation-data-01-58741847.us-east-1.elb.amazonaws.com`.

Here's a search to bring up the affected lines of code:
https://github.com/cityofaustin/transportation.austintexas.io/search?q=transportation-data.austintexas.io&unscoped_q=transportation-data.austintexas.io

### PR DESCRIPTION
**Pull request by tcwcodes**
Fixes #328

### CODE REVIEW

File: components/js/signal-timing.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ var SYSTEM_INTERSECTIONS_URL =
   "https://data.austintexas.gov/resource/efct-8fs9.json?$limit=5000";
 
 var LOGFILE_URL =
-  "https://transportation-data.austintexas.io/jobs?name=eq.signal_retiming&status=eq.success&order=start_date.desc&&records_processed=gt.0&limit=1";
+  "http://transportation-data-01-58741847.us-east-1.elb.amazonaws.com/jobs?name=eq.signal_retiming&status=eq.success&order=start_date.desc&&records_processed=gt.0&limit=1";
```
[the remark this answers is not in the document]
Comment (johnclary) [reply components/js/signal-timing.js]: @mateoclarke i believe I can fix this now. let me test

[the remark this answers is not in the document]
Comment (sergiogcx) [reply components/js/signal-timing.js]: @mateoclarke The load balancer needs the certificate of the domain it is associated to

[the remark this answers is not in the document]
Comment (sergiogcx) [reply components/js/signal-timing.js]: This should be easily fixable now that I look into it, all there is is to switch the SSL which is already installed

[the remark this answers is not in the document]
Comment (johnclary) [reply components/js/signal-timing.js]: @mateoclarke @tcwcodes yes, we can replace these with `https://api.mobility.austin.gov`. Just had to add the new cert to the ELB.

[the remark this answers is not in the document]
Comment (sergiogcx) [reply components/js/signal-timing.js]: @mateoclarke done! https://api.mobility.austin.gov/

[the remark this answers is not in the document]
Comment (sergiogcx) [reply components/js/signal-timing.js]: I've also replaced the endpoints, maybe it's good for a review now?

[the remark this answers is not in the document]
Comment (tcwcodes) [reply components/js/signal-timing.js]: Looks like Sergio beat me to it! Let me know if there's anything else I can do on this one.

**Pull request merged**